pilot/pkg/config/kube/ingress/conversion_test.go

```diff
@@ -21,10 +21,123 @@ import (
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 
+	networking "istio.io/api/networking/v1alpha3"
+
```

Comment:
remove blank line

Reply:
I'm running "make format" - I think manual management of imports (or spending time reviewing the order) is not best use of time, adding imports is also automated by the IDE.

Reply:
sure, i just happened to notice it.

Reply:
If linter complains - it'll have to be fixed. Otherwise - it's far more valuable to spend your time looking at efficiency/correctness of the code and the big picture...